admin/includes/admin_html_head.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * @package admin
+ * @copyright Copyright 2003-2016 Zen Cart Development Team
+ * @copyright Portions Copyright 2003 osCommerce
+ * @license http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
+ * @version $Id: admin_html_head.php  New in v1.5.7 $
+ */
+?>
+<meta charset="<?php echo CHARSET; ?>">
```

Comment:
Can we add a check to abort if `IS_ADMIN_FLAG` isn't defined. That prevents malicious actors from reaching in and outputting stuff that either reveals things it shouldn't or at least avoids filling error logs with notices about undefined vars etc.

Reply:
done